rules/otp.bzl

```diff
@@ -148,3 +148,47 @@ otp = rule(
         "lc_count": attr.int(doc = "Lifecycle count", default = 8),
     },
 )
+
+def _otp_image(ctx):
+    # TODO(dmcardle) I don't like hardcoding the width in the filename. Maybe we
+    # can write it into some metadata in the file instead.
+    output = ctx.actions.declare_file(ctx.attr.name + ".24.vmem")
+    args = ctx.actions.args()
+    args.add("--quiet")
+    args.add("--lc-state-def", ctx.file.lc_state_def.path)
+    args.add("--mmap-def", ctx.file.mmap_def.path)
+    args.add("--img-cfg", ctx.file.src.path)
+    args.add("--out", "{}/{}.BITWIDTH.vmem".format(output.dirname, ctx.attr.name))
+    ctx.actions.run(
+        outputs = [output],
+        inputs = [
+            ctx.file.src,
+            ctx.file.lc_state_def,
+            ctx.file.mmap_def,
+        ],
+        arguments = [args],
+        executable = ctx.executable._tool,
+    )
+    return [DefaultInfo(files = depset([output]), data_runfiles = ctx.runfiles(files = [output]))]
```

Comment:
I know you're just moving this code, but I think we can also `data_runfiles` here :)

Reply:
It turns out in this case, the runfiles parameter is necessary since the Verilator test that depends on this rule requires this file at runtime (i.e. the VMEM is passed as a parameter into the Verilator command). I've added it back as `runfiles = ctx.runfiles(...)` per the the [documentation's guidelines](https://bazel.build/extending/rules#runfiles_features_to_avoid)

Reply:
Wouldn't it make more sense for the Verilator test to define its own runfiles? Happy to consider it out of scope for this PR, you've already improved what was there :)